### ISSUE TITLE
Databricks provider, Error thrown on terraform plan -  Error: expected format to be one of [SOURCE], got DBC

### ISSUE THREAD

**Issue by jvandenbos** [datarootsio/terraform-module-azure-datalake#168]
When running terraform plan, terraform throws this error:

```
Error: expected format to be one of [SOURCE], got DBC

  on sample_data.tf line 31, in resource "databricks_notebook" "presentation":
  31: resource "databricks_notebook" "presentation" {
```

A temporary, but probably not the right, fix involves: In the section described in the error message above, changing the line that says:

  format    = "DBC"

to:

  format    = "SOURCE"

N.B.: Possibly related to issue #165

**Comment by sdebruyn** [comment]
Fixed in 0.4.13

**Comment by sdebruyn** [comment]
This used to be supported by the Databricks provider, but was recently removed. So it's now removed from this module as well.
